sdk/Lusid.Sdk/Model/LabelValueSet.cs

```diff
@@ -0,0 +1,114 @@
+ *
+ * The version of the OpenAPI document: 0.11.1667-GD
+ * Contact: info@finbourne.com
+ * Generated by: https://github.com/openapitools/openapi-generator.git
+ */
+
+using System;
+using System.Linq;
+using System.IO;
+using System.Text;
+using System.Collections;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Runtime.Serialization;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Converters;
+using OpenAPIDateConverter = Lusid.Sdk.Client.OpenAPIDateConverter;
+
+namespace Lusid.Sdk.Model
+{
+    /// <summary>
+    /// The set of string labels in a multi-value property.
+    /// </summary>
+    [DataContract]
+    public partial class LabelValueSet :  IEquatable<LabelValueSet>
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="LabelValueSet" /> class.
+        /// </summary>
+        /// <param name="values">values.</param>
+        public LabelValueSet(List<string> values = default(List<string>))
+        {
+            this.Values = values;
+            this.Values = values;
```

Comment:
I am assuming that this is automatically generated. With that in mind do you know why this.Values = values; appears in here twice?

Reply:
This is auto generated and occurs in multiple places. I haven't looked into why this is duplicated.